.fcrepo_wrapper

```diff
@@ -0,0 +1,10 @@
+# Place any default configuration for fcrepo_wrapper here
+port: 8984
+enable_jms: false
+
+download_dir: tmp/fcrepo_wrapper
+
```

Comment:
The blank lines look unsightly to me. Open to yanking those out? I don't think they aid readability or organization here.